File: 4way/problem002/solution.c
int largestAltitude(int* gain, int gainSize) {
    int max = 0;
    int prev = 0;
    for(int i = 0; i < gainSize; i++){
        prev += gain[i];
        max = max > prev ? max : prev;
    }

    return max;
}

File: 4way/problem002/solution.js
/**
 * @param {number[]} gain
 * @return {number}
 */
var largestAltitude = function(gain) {
    let max = 0, prev = 0

    gain.forEach((point) => {
        prev += point
        max = Math.max(max, prev)
    })

    return max
};
